keyboards/vitamins_included/keymaps/numpad/keymap.c

```diff
@@ -0,0 +1,81 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
+#include "eeconfig.h"
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _NUMPAD 0
+#define _ADJUST 3
+
+enum custom_keycodes {
+  NUMPAD = SAFE_RANGE,
+  ADJUST
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
```

Comment:
These two keycodes are already defined in QMK, there's no need to add them here :)